include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -306,7 +306,7 @@
 // when target devices are CPU or FPGA emulator. Since CPU and GPU devices
 // cannot be distinguished during compilation, the macro is enabled by default.
 #    if !defined(_ONEDPL_COMPILE_KERNEL)
-#        define _ONEDPL_COMPILE_KERNEL 1
+#        define _ONEDPL_COMPILE_KERNEL 0
```

Comment:
Why have we modified the default behavior?
(In any case the comment in the code before is not actual any more and should be updated)

Reply:
I believe this change is already rolled back